global.json

```diff
@@ -1,9 +1,9 @@
 {
   "sdk": {
-    "version": "3.1.302"
+    "version": "3.1.401"
   },
   "tools": {
-    "dotnet": "3.1.302",
+    "dotnet": "3.1.401",
     "vs": {
       "version": "16.4",
```

Comment:
If you're updating to `3.1.401`, then this line has to be updated to `16.7`, because `401` depends on that specifically.

You'll also need to run this against the internal signed build.  Like @jonsequitur mentioned, we had to back this change off due to the internal VMs lagging, but last week they were updated with 16.7.

Reply:
Okay, updated. Will it work now?